src/Console/Console.php

```diff
@@ -492,7 +492,7 @@ private function help($command = null)
 U;
                 break;
 
-            case 'run':
+            case 'run': // phpcs:disable
```

Comment:
The following characters are greater than the `120` lengths in the PSR-2 coding style.

To ignore this line, adding the `// phpcs:disable` comment.